packages/clay-pagination-bar/src/PaginationBarWithBasicItems.tsx

```diff
@@ -246,7 +246,7 @@ export const ClayPaginationBarWithBasicItems = ({
 						spritemap={spritemap}
 					>
 						{(item) => (
-							<Option key={item.label}>
+							<Option href={item?.href} key={item.label}>
```

Comment:
Do we need the `?`

Since it's not used for item.label it seem we wouldn't need it for item?.href

Reply:
I added it because there might not be a href property on the object.

Reply:
So that notation isn't needed for that case. `item?.href` is designed to handle the situation where `item` might be null or undefined. If `item` is null or undefined, then calling `item.href` will result in an error: "Cannot read properties of undefined|null". It's this situation that `item?.href` is for. This notation says if item is undefined or null, don't thrown an error, just return undefined. You can read more about this [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining).

Reply:
Ahh, you're right! Thanks, Evan, I'll remove it.